cmd/inband.go

```diff
@@ -135,8 +141,10 @@ func (i *inbandCmd) Exec(ctx context.Context, _ []string) error {
 		}
 
 		device.ID = i.assetID
-		device.Model = inventoryAsset.Model
-		device.Vendor = inventoryAsset.Vendor
+		if inventoryAsset != nil {
+			device.Model = inventoryAsset.Model
+			device.Vendor = inventoryAsset.Vendor
+		}
```

Comment:
Why the `inventoryAsset.Model` and `inventoryAsset.Vendor` are different when using an `inventoryAssert` and the `device.Model` and `device.Vendor` collected by `collector.InventoryLocal()`?

Reply:
Thanks for pointing this out, I went back and fixed a few things so its more clearer.

These attributes are set on the collected asset inventory which is passed to the publisher to make sure its not being overwritten in the inventory, you can see the explanation in this commit https://github.com/metal-toolbox/alloy/pull/56/commits/a609fd10f79be22353aad3e9266fe7d4b62e1d07